flashflood/objects.py

```diff
@@ -273,10 +273,17 @@ def keys(self):
         return [f"{self._journal_pfx}/{id_}", f"{self._blobs_pfx}/{id_.blob_id}"]
 
     @classmethod
-    def list(cls) -> typing.Iterator[JournalID]:
+    def list(cls, list_from: JournalID=None) -> typing.Iterator[JournalID]:
+        """
+        List the latest non-tombstoned version of each journal.
+        if `list_from` is provided, listing will begin after `list_from`.
```

Comment:
What do you mean by "from" with Journal IDs? What I mean is, if these were timestamps, from/to would refer to the natural order of time. Would be good to clarify that Journal IDs are time-based, or if not, clarify what ordering "from" refers to

Reply:
I was hoping that "list_from" would help indicate that the value of `list_from` would _not_ appear in the listing. This behavior is passed through from S3.

I agree that there needs to be documentation about the ordering expectations of journal ids. I'll get to that :)